app/View/Pages/series.ctp

```diff
@@ -0,0 +1,100 @@
+<?php $this->set('title_for_layout', Configure::read("Club.name")); ?>
```

Comment:
instead of duplicating the whole home page, could you pull out just the bits you want to change into "element"s?

For example, I imagine the existing logic could go into a new `Elements/Events/summary_list.ctp`:

```
<?php
        $events = array();
        $events["Ongoing"] = $this->element('Events/box-list', array('filter' => 'ongoing', 'limit' => '-1'));
        $events["Upcoming"] = $this->element('Events/box-list', array('filter' => 'upcoming', 'limit' => '4'));
        $events["Past"] = $this->element('Events/box-list', array('filter' => 'past', 'limit' => '2'));

        foreach($events as $title => $content) {
            if(!empty($content)) {
                echo "<h3>${title}</h3>";
                echo $content;
            }
        }
        ?>
```

And your new UI could go into `Elements/Events/summary_list_by_series.ctp`.

Then in `home.ctp`, just a simple conditional:

```
if (Configure::read('Club.layout') === 'series' ) {
    echo $this->element('Events/summary_list_by_series');
} else {
    echo $this->element('Events/summary_list');
}
```